lib/rosette/integrations/smartling-integration/smartling_completer.rb

```diff
@@ -62,6 +62,30 @@ def delete_file(file)
         end
 
         def update_logs(locales_map)
+          status = if all_locales_are_complete?(locales_map)
+            Rosette::DataStores::PhraseStatus::TRANSLATED
+          else
+            Rosette::DataStores::PhraseStatus::PENDING
+          end
+
+          commit_id = derive_commit_id_from(locales_map)
+
+          rosette_config.datastore.add_or_update_commit_log(
+            repo_config.name, commit_id, nil, status
+          )
+
+          update_locale_logs(locales_map)
+        end
+
+        def derive_commit_id_from(locales_map)
+          repo_config.locales.each do |locale|
+            if file = locales_map[locale.code]
```

Comment:
It's supposed to be assignment, the next line returns `file.commit_id`. If the result of the assignment is nil, the method looks at the other files in `locales_map`.